cmd/kube-proxy/app/server.go

```diff
@@ -150,12 +162,32 @@ func AddFlags(options *Options, fs *pflag.FlagSet) {
 	utilfeature.DefaultFeatureGate.AddFlag(fs)
 }
 
+func NewOptions() Options {
+	o := Options{
+		config:      new(componentconfig.KubeProxyConfiguration),
+		healthzPort: 10256,
+	}
+
+	var (
+		groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+		registry             = registered.NewOrDie("")
+	)
+
+	o.scheme = runtime.NewScheme()
```

Comment:
@sttts to be honest, I'm still learning the ins and outs of API registration, schemes, etc, so whatever you all say is the right way to do this, I'm happy to do that. So if we don't want or need a registry, I'll get rid of it.